cpp/ql/lib/semmle/code/cpp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -205,6 +205,8 @@ predicate clearsContent(Node n, Content c) {
  */
 predicate expectsContent(Node n, ContentSet c) { none() }
 
+predicate typeStrongerThan(DataFlowType t1, DataFlowType t2) { none() }
```

Comment:
It would be super helpful to have some QLDoc to explain what "strong" means in this context. Is it (roughly) equivalent to saying that `t1` is a strict subtype of `t2`?

Reply:
Yes. I postponed qldoc a bit, since I'll hopefully soon rewrite the language-to-library api to go through a signature, such that the entire library can be a parameterised module in a qlpack - at that point all these predicates will get language-agnostic qldoc in the signature.